tests/integration/targets/ios_vrf_interfaces/vars/main.yaml

```diff
@@ -1,7 +1,8 @@
 ---
 gathered:
   config:
-    - name: "Loopback24"
+    - name: "Loopback888"
```

Comment:
Are these being cleaned up later?

Reply:
This by default exists in downstream ci but not in cml hence making sure it exists by adding it!

Reply:
The other loopback999 is being removed

Reply:
Okay, so that means it's not getting cleaned up. We should address that separately.